### DISCUSSION TITLE
Official Solar Panel(s)

### DISCUSSION THREAD

**Discussion by baqwas** [PiSupply/PiJuice#1054] [Q&A] [answered]
The _hardware/README.md_ mentions official solar panels but I could not find any such list. I would like to use a solar panel rated at 6V 0.5 A, would the J4 connection on the PiJuice board be practical for this purpose? Thanks.

Regards.

**Comment by shawaj** [comment, marked as answer]
The "official" ones are all USB... You can see them all here https://uk.pi-supply.com/collections/pijuice

Having said that, the j4 is fine for a solar panel. Or you can use the j3 underneath it (which is usually the better option IMO)

**Reply by baqwas** [reply]
Thanks, @shawaj! Will follow your advice.

Regards.
